addons/website_mass_mailing/static/tests/tours/newsletter_popup.js

```diff
@@ -44,13 +44,12 @@ tour.register('newsletter_popup_use', {
 }, [
     {
         content: "Check the modal is not yet opened and force it opened",
-        trigger: '.o_newsletter_popup',
-        run: function (actions) {
-            const $modal = this.$anchor.find('.modal');
+        trigger: 'body:has(.o_newsletter_popup)',
+        run: () => {
+            const $modal = $('.o_newsletter_popup .modal');
```

Comment:
Careful that in 16.0/master, searching in the whole DOM won't work and this will be silently broken since the jquery style `if` will be false instead of crashing as it should.

Reply:
Yep, this was not strong enough anymore.
I added a check to ensure we found one (and only one) such modal, that will prevent the test to silently fail if wrong or no element is found